src/content/language/operators.md

```diff
@@ -262,10 +262,10 @@ an operation with an assignment.
 
 Here's how compound assignment operators work:
 
-|                                  | Compound assignment           | Equivalent expression            |
-|----------------------------------|-------------------------------|----------------------------------|
-| **For an operator <em>op</em>:** | <code>a <em>op</em>= b</code> | <code>a = a <em>op</em> b</code> |
-| **Example:**                     | `a += b`                      | `a = a + b`                      |
+|                           | Compound assignment | Equivalent expression |
+|---------------------------|---------------------|-----------------------|
+| **For an operator *op*:** | `a `  *`op`*  `= b` | `a = a ` *`op `* `b`  |
```

Comment:
I think this one still makes sense with no space before the `=` if possible to match the spacing of the actual operators:

```suggestion
| **For an operator *op*:** | `a `  *`op`*`= b`   | `a = a ` *`op `* `b`  |
```

Reply:
thank you for catching that!